usage/app/model/ContentDetails.scala

```diff
@@ -20,7 +24,15 @@ object ContentDetails {
     ContentDetails(
       content.webTitle,
       content.webUrl,
-      content.sectionId.getOrElse("none")
+      content.sectionId.getOrElse("none"),
+      composerUrl(content)
     )
   }
+
+  def composerUrl(content: Content): Option[URL] = content.fields
+    .flatMap(_.internalComposerCode)
+    .flatMap(composerId => {
+      Try(new URL(s"${Config.composerBaseUrl}/${composerId}")).toOption
```

Comment:
I'm not sure whether it makes sense to denormalise this in ES but _not_ in the metadata dynamo store. The composer base url is not necessarily constant but the ID is and can be used to extract further metadata from other systems (Workflow for example).